packages/utils/tests/npm.ts

```diff
@@ -10,6 +10,8 @@ type Fs = {
     existsSync: () => boolean;
 };
 
+type HasYarnLock = () => boolean;
```

Comment:
This should return a `Promise<boolean>`

Reply:
@molant Changed these but `hasYarnLock` still not properly stubbed.